scripts/export-facecap-onnx.py

```
@@ -0,0 +1,789 @@
+                          data=np.asarray(img.convert("RGB")))
+        res = ref.detect(mp_img)
+        mine = ours(img)
+        if not res.face_landmarks:
+            print(f"  {os.path.basename(path)}: no face (reference); "
+                  f"ours={'none' if mine is None else 'found'}")
+            continue
+        if mine is None:
+            print(f"  {os.path.basename(path)}: REFERENCE found a face, we did NOT")
+            continue
+        W, H = img.size
+        ref_pts = np.array([[l.x * W, l.y * H] for l in res.face_landmarks[0]])
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)